onnxruntime/python/tools/quantization/E2E_example_model/e2e_user_example.py

```diff
@@ -0,0 +1,81 @@
+import os
```

Comment:
we don't need calibrate_user.py and data_preprocess.py anymore as we already have this file.